conf/Config.js

```diff
@@ -31,33 +31,27 @@ class Config {
         const parsedConfig = joi.attempt(config, backbeatConfigJoi,
                                          'invalid backbeat config');
 
+        if (parsedConfig.extensions) {
+            Object.keys(parsedConfig.extensions).forEach(extName => {
+                const index = require(`../extensions/${extName}/index.js`);
```

Comment:
I recommend looping through `extensions` folder and require all extensions at the top.

Reply:
If just a style preference, I would rather leave it here as I like the fact that I can just browse a config validated by joi first, guaranteed to have the fields I need to browse.

If there are other good reasons to move `require` to the top (caching or other) I can do the necessary manual checks on the extensions array, then require extensions in a global array and then browse that array here to check each extension config.